### PR TITLE
Feuture/rbsfxirjtp 209 aurora client side service refactor class to be more generic

### PR DESCRIPTION
**Pull request by rusevradoslav**

### CODE REVIEW

File: common/pom.xml [hunk 1]
```diff
@@ -17,11 +17,28 @@
...
+
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-api</artifactId>
+        </dependency>
+
+        <dependency>
+            <groupId>io.grpc</groupId>
+            <artifactId>grpc-api</artifactId>
+            <version>1.36.0</version>
```
Comment (zaraandreevaa) [inline common/pom.xml:39]: could you please extract the version and add the dependency in dependency management

File: common/pom.xml [hunk 2]
```diff
@@ -17,11 +17,28 @@
...
     <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+        </dependency>
+
         <dependency>
             <groupId>com.google.code.gson</groupId>
             <artifactId>gson</artifactId>
             <version>2.8.6</version>
```
Comment (zaraandreevaa) [inline common/pom.xml:28]: please refactor also this version

**Review by radevdetelin — APPROVED** [review]

File: calculator/src/main/java/com/market/banica/calculator/service/ProductServiceImpl.java [hunk 3]
```diff
@@ -196,11 +200,21 @@ private Product getProductFromDatabase(String productName) {
...
         LOGGER.debug("In writeProductToDatabase private method");
 
+        announceInterestToOrderBookProductBase(newProductName);
+
         productBase.getDatabase().put(newProductName, newProduct);
 
         backUpService.writeBackUp();
     }
 
+    private void announceInterestToOrderBookProductBase(String newProductName) {
```
Comment (gaantonova) [inline calculator/src/main/java/com/market/banica/calculator/service/ProductServiceImpl.java:210]: this method is part of another PR, why did you add it here as well?
